store/src/sql/entities/event.rs

```diff
@@ -87,20 +88,129 @@ impl ReconHash {
     }
 }
 
-#[derive(Debug, Clone, sqlx::FromRow)]
-pub struct EventValueRaw {
+use anyhow::anyhow;
+use multihash::{Code, MultihashDigest};
+
+#[derive(Debug, Clone, PartialEq, Eq, Hash, sqlx::Type)]
+pub struct BlockHash(Multihash);
```

Comment:
@stbrody we could do something similar for CID if we wanted to impl sqlx::Type to indicate it should be blob and how to use it in both `.bind()` and `query_as` FromRow responses.